code/game/objects/effects/landmarks.dm

```diff
@@ -4,11 +4,11 @@
 	icon_state = "x2"
 	anchored = 1
 	unacidable = 1
+	invisibility = INVISIBILITY_ABSTRACT
 
 /obj/effect/landmark/New()
 	..()
 	tag = text("landmark*[]", name)
-	invisibility = INVISIBILITY_ABSTRACT

```

Comment:
I went through this very line of thought.

I think it's just because landmarks are SUPER old